ramalama/cli.py

```diff
@@ -52,8 +52,63 @@ def add_argument(self, *args, help=None, default=None, **kwargs):
         super().add_argument(*args, **kwargs)
 
 
+def parse_config_file(file_path, config):
```

Comment:
Lets move the config parsing to a separate file config.py?

Isn't there a standard toml parser written in Python that we can use?

Reply:
```
import tomllib

with open("/usr/share/containers/containers.conf", "rb") as f:
    data = tomllib.load(f)

print(data)

```

Reply:
tomllib is available since 3.11, not available on all platforms.